### PR TITLE
Redirect admin to dashboard after invitation accept

### PR DESCRIPTION
**Pull request by Tomaszabc**
# 🔗 Issue
https://github.com/rubyforgood/pet-rescue/issues/264

# ✍️ Description
Redirect admin to dashboard after invitation accept

# 📷 Screenshots/Demos

https://github.com/rubyforgood/pet-rescue/assets/131488886/6829a74f-0a04-4057-aaf8-2935764c1bd1

### CODE REVIEW

File: app/controllers/organizations/invitations_controller.rb [hunk 1]
```diff
@@ -29,4 +29,8 @@ def user_params
       .permit(:first_name, :last_name, :email,
         staff_account_attributes: [:roles])
   end
+
```
Comment (kasugaijin) [inline app/controllers/organizations/invitations_controller.rb:32]: Nice. So this method is overriding the same method in `DeviseInvitable::Controllers::Helpers` ?

**Comment by kasugaijin** [comment]
@all-contributors
please add @Tomaszabc for code.

**Review by kasugaijin — APPROVED** [review]
LGTM.

File: app/controllers/organizations/invitations_controller.rb (hunk 1 above)
Comment (Tomaszabc) [reply app/controllers/organizations/invitations_controller.rb:32]: Yes. This method override the same method existing in Devise Invitable gem. I found method name "after_sign_out_path_for(resource_name)" in Devise Invitable documentation.

Comment (kasugaijin) [reply app/controllers/organizations/invitations_controller.rb:32]: Ah nice find!

**Pull request merged**